MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```

Comment:
- [OPTIONAL] Since this text is just a guide on how to customize your MIT please read it carefully and ensure to remove it after that 👍🏼